docs_src/design/ucr/Provision-Watch-via-Device-Metadata.md

```diff
@@ -0,0 +1,102 @@
+- Device Owner
+- Device User
+- Device Maintainer
+- Cloud User
+- Service Provider
+- Software Integrator
+
+### Description
+Application Services using the Device SDKs (ie, Hybrid App-Device Services) can take advantage of the
+Provision Watching feature and APIs to "discover" new EdgeX devices from the south-bound Device 
+Services, match them to app-specific Device Profiles, and handle their data with analysis or transforms.
+
+A south-bound Device Service may discover devices across a range of protocol properties, and those devices may need different
+Device Profiles depending upon metadata properties of the discovered devices, for example, the ModelName field of BACnet data. 
+While the "modelName" is an obvious target, the Device Service may want to use other device metadata for Provisioning 
+as well for inclusion or exclusion.
+
+For another example, consider the case where each of three Hybrid App-Device Services (a Trending Service, an Alarm Monitoring Service, and a Cloud Service) want to handle the data originating in a south-bound
+Modbus service for any "Watt-o-Meter" (Model Name) Device. So each service is configured with a Provision Watcher that will try to match that "modelName", or else "profileName" of "Watt-o-Meter-Modbus-Profile-01", of devices discovered in core-metadata
+or shown as added via the control plane events and, if a match is found, add a new "extended" Device to each service using the appropriate Device Profile (eg, "Watt-o-Meter-Trends-Profile-01" for the Trending Service), and giving the new extended Device a name, for example based on the original and the service (eg, "Meter-333-Trending").
+
+### Extensions
+Other Device metadata properties appear to be good candidates for a user to choose from:
+- name: The Device name may be good for regular expression matching, eg "name":"Meter-*"
+- labels: Since this one is free-form and open to the owner to add labels of their choosing, 
+this one should be good for both matching and the exclusion list. 
+Eg, if a Device had "labels": "meter, basement, energy", then it could be matched or excluded for "labels":"basement".
+- serial number: with regular expressions, this can be a powerful matching choice.
+- MAC address: similar to serial number for a specific range of vendor devices.
+
+The Device Service which discovers the Device will probably want to permit specific metadata properties to be used.
+
+### Existing solutions
+In EdgeX today, as noted, Provision Watchers match only the protocol properties, using regular expression matching and excluding. The example given for the REST API is a good one:
+```json
+"identifiers": {
+        "address": "localhost",
+        "port": "3[0-9]{2}"
+      },
+```
+Note its use of regular expression matching for the port number. 
+The documentation should be clearer that the key-value pairs for "identifiers" and "blockingidentifiers" are only from the protocol properties, and that regular expression matching is supported.
```

Comment:
Line 81 should be filed as an issue on the documentation. I don't see its relevance here.

Reply:
OK, removed.